<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1596</width>
    <height>1038</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
	background-color: #fff;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QFrame" name="email_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLineEdit" name="email_input">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>374</width>
             <height>46</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>E-mail</string>
           </property>
          </widget>
          <widget class="QLabel" name="email_label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>311</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>16</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QFrame" name="password_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="password_label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>301</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Senha</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="password_input">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>374</width>
             <height>46</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Senha</string>
           </property>
          </widget>
          <widget class="QLabel" name="forgot_password">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>161</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>11</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">#forgot_password::hover {
	color: #389BF2;
}</string>
           </property>
           <property name="text">
            <string>Esqueceu a senha?</string>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="brand_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>180</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QLabel" name="greetings">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Bem-vindo(a)!</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="brand_name">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>48</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>PetCare</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>390</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>351</width>
            <height>120</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="login_button">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>1111</width>
               <height>1111</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">#login_button {
	background-color: #FFC000;
	color: #fff;
	border-radius: 5px;
}

#login_button:hover {
    background-color: #D89E00; 
}</string>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="register_label">
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>11</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">#register_label::hover {
	color: #389BF2;
}</string>
                </property>
                <property name="text">
                 <string>Não possui conta? Cadastre-se</string>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="validation_text">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>361</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#validation_text {
	color: #DE0A0C;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QFrame" name="frame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QProgressBar" name="loading_bar">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>350</width>
             <height>24</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>110</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QFrame" name="petcare_img">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>1000</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>1000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="img">
           <property name="minimumSize">
            <size>
             <width>781</width>
             <height>971</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>10000</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;resources\images\PetCareLogin.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>email_input</tabstop>
  <tabstop>password_input</tabstop>
  <tabstop>login_button</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/PetCareConnect.qrc"/>
 </resources>
 <connections/>
</ui>
